habitat_baselines/common/utils.py

```diff
@@ -228,6 +228,61 @@ def generate_video(
         )
 
 
+def tensor_to_depth_images(tensor: Union[torch.Tensor, List]) -> np.ndarray:
+    r"""Converts tensor (or list) of n image tensors to list of n images.
+    Args:
+        tensor: tensor containing n image tensors
+    Returns:
+        list of images
+    """
+    images = []
+
+    for img_tensor in tensor:
+        image = img_tensor.permute(1, 2, 0).cpu().numpy() * 255
+        images.append(image)
+
+    return images
+
+
+def tensor_to_bgr_images(tensor: torch.Tensor) -> List[np.ndarray]:
+    r"""Converts tensor of n image tensors to list of n BGR images.
+    Args:
+        tensor: tensor containing n image tensors
+    Returns:
+        list of images
+    """
+    import cv2
+
+    images = []
+
+    for img_tensor in tensor:
+        img = img_tensor.permute(1, 2, 0).cpu().numpy() * 255
+        img = img.astype(np.uint8)
+        img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)
+        images.append(img)
+
+    return images
+
+
+def get_q_string(question: List, q_vocab_dict: Dict) -> str:
```

Comment:
Can we use VocabDict's function here:

https://github.com/facebookresearch/habitat-lab/blob/89cf9842259c219418c9ad185bc2d26e7d1e68c4/habitat/datasets/utils.py#L85

Maybe even add a method `token_idx_2_string` to VocabDict will be more re-usable. Then you can call `q_vocab_dict.token_idx_2_string(question) + "?"` instead  `get_q_string(question, q_vocab_dict)`

Reply:
Thanks @mathfac. 

I'm removing all VQA related utils from this PR. I'll address this in a subsequent PR for the VQA baseline implementation.